### PR TITLE
Add script to Docker image to normalize bam index naming.

### PR DESCRIPTION
**Pull request by adthrasher**
Add RNAIndel.sh script to normalize BAM index names.
Swap CWL implementation to JSON.
Pin scikit-learn version due to `sklearn.ensemble.forest` rename.

### CODE REVIEW

**Review by jordan-rash — APPROVED** [review]
LGTM
